docs/tutorials/ServerModule/ServerModule.md

```diff
@@ -154,35 +164,21 @@ public class ModuleConfig : ConfigBase
 }
 ````
 
-## The ModuleConsole.cs - File <a id="moduleConsoleFile" />
+## The ModuleConsole.cs - File
 
-The module console provides a command line interface to interact with the server module without any custom client. It can be used for initial testing, debugging or 'admin access'-features. It can be used for initial testing, debugging or 'admin access'-features. As a starting point for this feature you can create ModuleConsole.cs file in your _ModuleController_ folder and copy the following code to it:
+The module console provides a way to execute methods using the maintenance. It can be used for initial testing, debugging or 'admin access'-features.As a starting point for this feature you can create ModuleConsole.cs file in your _ModuleController_ folder, implement `IServerModuleConsole` and add methods using the Attribute `EntrySerialize` in order to see them on the UI. Although the interface is empty, it's needed for the export.
```

Comment:
- ...features . As...
- ...for this feature you need to create a ModuleConsole.cs file ...